service/controller/aws/v10/key/key.go

```diff
@@ -21,6 +21,15 @@ func ChartSpecs() []key.ChartSpec {
 			ReleaseName:       "coredns",
 			UserConfigMapName: "coredns-user-values",
 		},
+		{
+			AppName:           "cluster-autoscaler",
+			ChannelName:       "0-1-stable",
+			ChartName:         "kubernetes-cluster-autoscaler-chart",
+			ConfigMapName:     "cluster-autoscaler-values",
```

Comment:
Do I need to set this somewhere else? I am not sure about this. Maybe @rossf7 can help?

Reply:
We talked today already. Just for the record. I think it is rather unfortunate to have these "specs" in the key package. I am not fully into the whole picture though, but it would be nice to keep the key packages as dumb as possible and get a clearer vision of how the chart CRs can be maintained.

Reply:
@MarcelMue Sorry I missed the ping here. :(

This is correct because you just want to enable on AWS. Once the chart is enabled for all providers it should move to the `key` package under `pkg`.

The idea when I refactored this was to have just the "specs" as structs in the key package. This is an intermediate step until we have proper config management. I'd like to see this in a yaml file like we have for the releases index.

Reply:
Yes, let's work into that direction. I did not see a benefit in doing this in the very PR here. Thanks for the heads up ross!